specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-10-01-preview/containerInstance.json

```diff
@@ -0,0 +1,977 @@
+          "description": "The name of the operation."
+        },
+        "display": {
+          "description": "The display information of the operation.",
+          "type": "object",
+          "properties": {
+            "provider": {
+              "description": "The name of the provider of the operation.",
+              "type": "string"
+            },
+            "resource": {
+              "type": "string",
+              "description": "The name of the resource type of the operation."
+            },
+            "operation": {
+              "description": "The friendly name of the operation.",
+              "type": "string"
+            },
+            "description": {
+              "description": "The description of the operation.",
+              "type": "string"
+            }
+          }
+        },
+        "origin": {
+          "type": "string",
+          "description": "The intended executor of the operation.",
+          "enum": [
+            "User",
+            "System"
+          ],
+          "x-ms-enum": {
+            "name": "ContainerInstanceOperationsOrigin",
+            "modelAsString": true
+          }
+        }
+      }
+    },        
+    "ContainerGroupListResult": {
+      "description": "The container group list response that contains the container group properties.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerGroup"
+          },
+          "description": "The list of container groups."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of container groups."
+        }
+      }
+    },
+    "Logs": {
+      "description": "The logs.",
+      "type": "object",
+      "properties": {
+        "content": {
+          "type": "string",
+          "description": "The content of the log."
+        }
+      }
+    },
+    "Resource": {
+      "type": "object",
+      "description": "The Resource model definition.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource id."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type."
+        },
+        "location": {
+          "type": "string",
+          "description": "The resource location."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The resource tags."
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
```

Comment:
Mark subscriptionID and APIversion with `"x-ms-parameter-location": "client"`

Reply:
Added.